### PR TITLE
AVX-14410: support upgrade to build

### PR DESCRIPTION
**Pull request by kzhangavtx**
Allow restore to major.minor-apha.build (e.g. 6.5-patch.100). Previously it was only allowed to restore to major.minor-apha or major.minor (e.g. 6.5-patch or 6.5)

Test results:
2021-12-02T16:04:34.007-08:00	Parsing version
2021-12-02T16:04:34.007-08:00	Parsed version sucessfully 6.5-patch and 6.5-patch.2771
2021-12-02T16:04:34.008-08:00	Checking if upgrade to build is suppported
2021-12-02T16:04:34.169-08:00	b'{"return":false,"reason":"Valid action required: get_feature_info"}'
2021-12-02T16:04:34.169-08:00	Upgrade to build is not supported
2021-12-02T16:04:34.169-08:00	Checking initial setup

### CODE REVIEW

File: aviatrix_ha.py [hunk 1]
```diff
@@ -504,6 +506,25 @@ def verify_backup_file(controller_instanceobj):
         return True, s3_file
 
 
+def support_upgrade_to_build(ip_addr, cid):
```
Comment (arvindsree) [inline aviatrix_ha.py:509]: You could rename to `is_upgrade_to_build_supported`

File: aviatrix_ha.py [hunk 2]
```diff
@@ -504,6 +506,25 @@ def verify_backup_file(controller_instanceobj):
...
+    """ Check if the version supports upgrade to build """
+    print("Checking if upgrade to build is suppported")
+    base_url = "https://" + ip_addr + "/v1/api"
+    post_data = {"CID": cid,
+                 "action": "get_feature_info"}
+    try:
+        response = requests.post(base_url, data=post_data, verify=False)
+        print(response.json())
```
Comment (arvindsree) [inline aviatrix_ha.py:517]: json decode could fail. you could do json.loads(response.content) and print response.content if it fails with ValueError. 
Also check for return Value. if return is False print the reason.

File: aviatrix_ha.py (hunk 1 above)
Comment (kzhangavtx) [reply aviatrix_ha.py]: Changed

File: aviatrix_ha.py (hunk 2 above)
Comment (kzhangavtx) [reply aviatrix_ha.py]: Changed to use response.content and handled exceptions

**Review by arvindsree — APPROVED** [review]

**Pull request merged**